cg/clients/janus/dto/create_qc_metrics_request.py

```diff
@@ -29,5 +16,5 @@ class CreateQCMetricsRequest(BaseModel):
     case_id: str
     sample_ids: list[str]
     files: list[FilePathAndTag]
-    workflow: SupportedWorkflow
-    prep_category: SupportedPrepCategory | None
+    workflow: str
+    prep_category: str
```

Comment:
So these should be allowed any value? Not only Workflow or PrepCategory or whatever the constants are called?